functions.opengraph.php

```diff
@@ -152,16 +152,17 @@ function jetpack_og_tags() {
 			$tags['og:description'] = wp_kses( trim( convert_chars( wptexturize( $tags['og:description'] ) ) ), array() );
 		}
 
-		$tags['article:published_time'] = date( 'c', strtotime( $data->post_date_gmt ) );
-		$tags['article:modified_time']  = date( 'c', strtotime( $data->post_modified_gmt ) );
+		$tags['article:published_time'] = gmdate( 'c', strtotime( $data->post_date_gmt ) );
+		$tags['article:modified_time']  = gmdate( 'c', strtotime( $data->post_modified_gmt ) );
 		if ( post_type_supports( get_post_type( $data ), 'author' ) && isset( $data->post_author ) ) {
 			$publicize_facebook_user = get_post_meta( $data->ID, '_publicize_facebook_user', true );
 			if ( ! empty( $publicize_facebook_user ) ) {
 				$tags['article:author'] = esc_url( $publicize_facebook_user );
 			}
 		}
+	} elseif ( is_search() ) {
+		$tags['og:title'] = 'Search results: ' . get_query_var( 's', '(no title)' );
```

Comment:
I think it would be better if this could be translatable.

I am also not sure I would add that "no title" default; if there are no keywords (i.e. if someone shares `mysite.com/?s=` on Facebook, maybe we shouldn't add a custom Open Graph title at all, since folks would most likely except to see the site title as their preview on Facebook in those cases. What do you think?